tests/rbd/rbd_system.py

```diff
@@ -15,7 +13,9 @@ def run(**kw):
     client_node = rgw_client_nodes[0]
     # cleanup any existing stale test dir
     test_folder = 'rbd-tests'
-    client_node.exec_command(cmd='rm -rf ' + test_folder)
+    client_node.exec_command(cmd='sudo yum -y install python3')
+    client_node.exec_command(cmd='python3 -m venv ' + test_folder)
+    client_node.exec_command(cmd='sudo rm -rf ' + test_folder)
```

Comment:
both these statements are contradictory. you are creating virtualenv with rbd-tests and deleting the folder in the next line. And you are still not activating virtualenv, you need to activate it and then install the packages. right now the packages are getting installed in system installed version of python. can you correct these